plugins/homeuserdata.lua

```diff
@@ -0,0 +1,76 @@
+local core = require "core"
+local command = require "core.command"
+local CommandView = require "core.commandview"
+
+-- don't execute plugin on windows
+if package.cpath:match("%p[\\|/]?%p(%a+)") == "dll" then
+  return
+end
+
+-- check if home directory is set
+local home_dir = os.getenv("HOME")
+
+if not home_dir then
+  return
+end
+
+-- user configurations path in order of preference
+local configs = {
+  home_dir .. "/.config/lite",
+  home_dir .. "/.lite"
+}
+
+-- load user settings
+for i,dir in ipairs(configs) do
+  local redirect = " 2>/dev/null"
+  if system.get_file_info(dir) then
+    package.path = package.path .. ";" .. dir .. "/?/init.lua"
+    package.path = package.path .. ";" .. dir .. "/?.lua"
+
+    if system.get_file_info(dir .. "/init.lua") then
+      local init = loadfile(dir .. "/init.lua")
+      init()
+    end
+
+    break
+  end
+end
+
+command.add(nil, {
+  ["core:open-home-user-module"] = function()
+    local directory = nil
+    for i,dir in ipairs(configs) do
+      local redirect = " 2>/dev/null"
```

Comment:
Is this a stray?

Reply:
Ahh since I didn't notice that system.get_file_info() existed I was using a io.pipe to check if a file existed with ls and was using that to hide the output. Thanks for catching this. I removed it.